packages/webdriver/src/index.js

```diff
@@ -37,6 +38,14 @@ export default class WebDriver {
              */
             : [{ alwaysMatch: params.capabilities, firstMatch: [{}] }, params.capabilities]
 
+        w3cCaps.alwaysMatch = JSON.parse(JSON.stringify(w3cCaps.alwaysMatch))
```

Comment:
Can we have it directly in the object creation? like
```js
/**
 * otherwise assume they passed in jsonwp-style caps (flat object)
 */
{ alwaysMatch: Object.keys(params.capabilities)
    .filter(key => !W3C_ALLOWED_CAPABILITIES.includes(key) && !key.match(/^[\w-]+:.*$/))
    .reduce((obj, key) => {
      obj[key] = params.capabilities[key]
      return obj
    }, {}), firstMatch: [{}] },
  params.capabilities]
```
instead of overwriting it afterwards.